### PR TITLE
Added step-by-step instructions to 'how-to-use-package.json-the-core-of-any-node.js-project-or-npm-package/index.md'

### PR DESCRIPTION
**Pull request by InaSLew**
Added step-by-step instructions to set up an anonymous node/express project for the challenges and three demo screenshots.

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

**Review by thecodingaviator — DISMISSED** [review]
Hi, thank you for your PR but now we are no longer accepting links or other resources (images, etc.) which are not under our control, i.e. which a user or service might remove and is not under our control.

I'd recommend removing the images or using alternatives

**Comment by InaSLew** [comment]
Ah I see.  The screenshots have been removed 😃

**Comment by AshutoshKumar9284** [comment]
DFSA

**Comment by AshutoshKumar9284** [comment]
dfghj

**Comment by AshutoshKumar9284** [comment]
zxcvbn

**Comment by AshutoshKumar9284** [comment]
no comments

**Comment by AshutoshKumar9284** [comment]
FGDESA

**Comment by AshutoshKumar9284** [comment]
FGDESA

**Comment by AshutoshKumar9284** [comment]
HGFDS

**Comment by Dharmin2108** [comment]
//

**Comment by Dharmin2108** [comment]
5

**Comment by Dharmin2108** [comment]
//

**Comment by Dharmin2108** [comment]
//

**Comment by Dharmin2108** [comment]
//

**Comment by Dharmin2108** [comment]
//

**Comment by Dharmin2108** [comment]
..

**Comment by Dharmin2108** [comment]
hii

File: guide/english/certifications/apis-and-microservices/managing-packages-with-npm/how-to-use-package.json-the-core-of-any-node.js-project-or-npm-package/index.md [hunk 1]
```diff
@@ -3,14 +3,53 @@ title: How to Use package.json, the Core of Any Node.js Project or npm Package
...
+
+ ❗️ ❗️ ❗️ ❗️ ❗️ ❗️ ❗️ ❗️ ❗️ ❗️ ❗️ ❗️  SPOILER ALERT. DO NOT SCROLL DOWN IF YOU WISH TO KEEP TRYING ❗️ ❗️ ❗️ ❗️ ❗️ ❗️ ❗️ ❗️ ❗️ ❗️ ❗️ ❗️ ❗️
+ 
+ 
+ 
+ 
+ 
+ 
+ 
+ ```json
```
Comment (thecodingaviator) [inline guide/english/certifications/apis-and-microservices/managing-packages-with-npm/how-to-use-package.json-the-core-of-any-node.js-project-or-npm-package/index.md]: Change the solution to two-step spacing and use the detail-summary pair of tags for the solution

**Review by thecodingaviator — DISMISSED** [review]

File: guide/english/certifications/apis-and-microservices/managing-packages-with-npm/how-to-use-package.json-the-core-of-any-node.js-project-or-npm-package/index.md (hunk 1 above)
Comment (RandellDawson) [reply guide/english/certifications/apis-and-microservices/managing-packages-with-npm/how-to-use-package.json-the-core-of-any-node.js-project-or-npm-package/index.md]: @thecodingaviator I fixed the spacing on behalf of the user.

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**